craftassist/agent/craftassist_agent.py

```diff
@@ -136,14 +136,27 @@ def send_text_command_to_agent(sid, command):
             )  # the chat is coming from a player called "dashboard"
             self.dashboard_chat = agent_chat
             dialogue_manager = self.dialogue_manager
-            # send back the dictionary
+            logical_form = {}
+            status = ""
             try:
-                x = dialogue_manager.get_logical_form(s=command, model=dialogue_manager.model)
-                logging.info("logical form is : %r" % (x))
-                payload = {"status": "Sent successfully", "chat": command, "chatResponse": x}
+                logical_form = dialogue_manager.get_logical_form(
+                    s=command, model=dialogue_manager.model
+                )
+                logging.info("logical form is : %r" % (logical_form))
+                status = "Sent successfully"
             except:
                 logging.info("error in sending chat")
-                payload = {"status": "Error in sending chat", "chat": command, "chatResponse": {}}
+                status = "Error in sending chat"
+            # update server memory
+            self.dashboard_memory["chatResponse"][command] = logical_form
```

Comment:
is it time to move this to loco_mc_agent? is that feasible?

Reply:
Sorry should've put that in the description - that's the plan. Planning to do that in a followup PR (since that would require refactoring both agent files) once this unblocks the bug on Minecraft.